docs/admin/resource-quota.md

```diff
@@ -31,12 +31,30 @@ Documentation for other releases can be found at
 
 <!-- END MUNGE: UNVERSIONED_WARNING -->
 
-# Administering Resource Quotas
+# Resource Quotas
 
-Kubernetes can limit both the number of objects created in a namespace, and the
-total amount of resources requested by pods in a namespace.  This facilitates
-sharing of a single Kubernetes cluster by several teams or tenants, each in
-a namespace.
+When several users or teams share a cluster with a fixed number of nodes,
+there is a concern that one team could use more than its fair share of resources.
+
+Resource quotas are a tool for administrators to address this concern. Resource quotas
+work like this:
+- Different teams work in different namespaces.  Currently this is voluntary, but
+  support for making this mandatory via ACLs is planned.
+- Users put [compute resource limits](../user-guide/compute-resources.md) on their pods.
+- The administrator creates a Resource Quota for each namespace.
+- If users exceed the limits specified in the Resource Quota in a namespace, they are blocked
```

Comment:
I would clarify this as "If a new Pod would exceed the limits specified in the Resource Quota in a namespace, then creation of the Pod is blocked."